src/main/java/de/tum/in/www1/artemis/domain/GuidedTourSettings.java

```diff
@@ -0,0 +1,109 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.io.Serializable;
+
+import com.google.gson.Gson;
+
+/**
+ * GuidedTourSettings
+ */
+public class GuidedTourSettings implements Serializable {
+
+    private boolean showCourseOverviewTour;
+
+    private boolean showNavigationTour;
+
+    private boolean showProgrammingExerciseTour;
+
+    private boolean showQuizExerciseTour;
+
+    private boolean showModelingExerciseTour;
+
+    private boolean showTextExerciseTour;
+
+    public GuidedTourSettings() {
+        this.showCourseOverviewTour = false;
+        this.showNavigationTour = false;
+        this.showProgrammingExerciseTour = false;
+        this.showQuizExerciseTour = false;
+        this.showModelingExerciseTour = false;
+        this.showTextExerciseTour = false;
+    }
+
+    public GuidedTourSettings(boolean showCourseOverviewTour, boolean showNavigationTour, boolean showProgrammingExerciseTour, boolean showQuizExerciseTour,
```

Comment:
So for every tour we need a new paramter in the constructor?
I think this should be better something that is stored in a Map or so :thinking:

Reply:
@thilo-behnke I noticed that I don't need this constructor yet (it was implemented in thoughts of possible future use of it). But since there is no need for it now, I will just delete it (and implement it rightfully if it is needed in future).